DEPLOYNOTES.rst

```diff
@@ -3,6 +3,11 @@
 Deploy and Upgrade notes
 ========================
 
+0.23
+----
+
+* This release requires a reindex to update the gender field for use in faceting.
```

Comment:
Thanks for noting this. We probably will have other solr changes for v0.23, but might be worth noting that we can run ` python manage.py index -i person` to just index the people, since indexing the books takes longer and is unnecessary.